### ISSUE TITLE
Make ability to send inscriptions more obvious

### ISSUE THREAD

**Issue by markmhendrickson** [blockstack/blockstack-app#3509]

**Comment by kyranjamie** [comment]
Curious to learn the thinking behind this suggestion. Then we add _n times the number of inscriptions_ send buttons to the UI, distracting from the content.

**Comment by markmhendrickson** [comment]
Users don't always realize it's there according to @hozzjss and @314159265359879

**Comment by mica000** [comment]
@314159265359879 I agree with Kyran that we might create too much disturbance on the home. Let's try to avoid cluttering information that its not primary. Sending an Ordinal should be achieved in 2 ways, via the card and via the send button (which was not built yet).

Would be possible to gather a bit more insight into this issue? How many times have you seen this occurring? Were they managed to find it eventually?

This would be great to know. 
Thank you very much!
